### ISSUE TITLE
Unable to link app as a service

### ISSUE THREAD

**Issue by mpapillon** [CleverCloud/clever-tools#320]
When I try to run :

```
clever service link-app -a my-alias my-app
```

I have an error described below. Despite, my apps are linked to the CLI.

```
[ERROR] Several applications are linked. You can specify one with the "--alias" option. Run "clever applications" to list linked applications.
```

**Comment by hsablonniere** [comment]
Thanks very much @mpapillon :+1:

**Comment by hsablonniere** [comment]
@mpapillon Your fix is in 1.4.2. Thanks :heart_eyes:

**Comment by mpapillon** [comment]
Thanks for the quick release 👌
